Makefile

```diff
@@ -432,7 +432,6 @@ compress-tar:
 	./apisix \
 	./bin \
 	./conf \
-	./apisix-$(VERSION)*.rockspec \
```

Comment:
why remove it？

Reply:
because rockspecs have been deleted: https://github.com/apache/apisix/commit/a44091243fb61cc9e1b3598c7d1218ca2470bd3b

and it causes CI failure if we don't remove this line.